libraries/lib-xml/XMLWriter.h

```diff
@@ -56,9 +56,10 @@ class XML_API XMLWriter /* not final */ {
 
    virtual void Write(const wxString &data) = 0;
 
+ private:
    // Escape a string, replacing certain characters with their
    // XML encoding, i.e. '<' becomes '&lt;'
-   static wxString XMLEsc(const wxString & s);
+   wxString XMLEsc(const wxString & s);
```

Comment:
I'm not sure if I understand why static was removed. I do not see any changes to the function, so I see little reason why it is no longer static or at least why is it not const.

Reply:
It isn't static in 3.1.  Static was inserted by @lllucius in the prototype he left us only to implement new effect stacks, and I cherry-picked that part months ago.  Now with better understanding, I see it as unnecessary for correct implementation.

Reply:
You miss my point. Does this function requires `this`? Does this function modifies `this`? At least it should be `const`, but I would prefer it to be `static`. In this case it can be removed from the class altogether.

Reply:
Okay, I'll leave it as static, but I think it should be private too.

Reply:
I have no objections on `private`